### PR TITLE
feat: add offset and limit to listimports

### PR DESCRIPTION
**Pull request by peterdeme**
- Add Offset and Limit to import endpoints

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        goVer: ["^1.16", "^1.17"]
+        goVer: ["1.16", "1.17"]
```
Comment (peterdeme) [inline .github/workflows/ci.yml:19]: image

Comment (ferhatelmas) [reply .github/workflows/ci.yml:19]: yes, it uses latest by default, previous change was unnecessary

File: import.go [hunk 2]
```diff
@@ -65,9 +72,15 @@ func (c *Client) GetImport(ctx context.Context, id string) (*GetImportResponse,
 // Note: Do not use this.
 // It is present for internal usage only.
 // This function can, and will, break and/or be removed at any point in time.
-func (c *Client) ListImports(ctx context.Context) (*ListImportsResponse, error) {
+func (c *Client) ListImports(ctx context.Context, opts *ListImportsOptions) (*ListImportsResponse, error) {
```
Comment (ferhatelmas) [inline import.go:75]: it doesn't matter here but in general, we should have options pattern as a habit/consistency

`ListImports(ctx, WithLimit(10)) // offset is zero` 
`ListImports(ctx, WithLimit(20), WithOffset(10))`

**Review by ferhatelmas — APPROVED** [review]

**Comment by ferhatelmas** [comment]
ci is updated :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (peterdeme) [reply .github/workflows/ci.yml:19]: yup, my bad.

can you remove these checks?
image

**Pull request merged**